cli/lib/kontena/cli/stacks/yaml/stack_file_loader.rb

```diff
@@ -27,7 +27,9 @@ def self.loaders
       # @return [StackFileLoader]
       def self.for(source, parent = nil)
         loader = loaders.find { |l| l.match?(source, parent) }
-        raise "Can't determine stack file origin for '#{source}'" if loader.nil?
+        if loader.nil?
+          raise RuntimeError, "Not found: no such file #{source} or invalid uri scheme"
```

Comment:
perhaps could be `Errno::ENOENT`

Reply:
The `RuntimeError : Not found: no such file ... or invalid uri scheme` probably isn't any less weird than `Errno::ENOENT : No such file or directory @ rb_sysopen`. Ideally this would exit with some even friendlier error as opposed to the DEBUG= etc hinting for unexpected errors, but this is fine.

### 1.3.4
```
/kontena/test # kontena stack install foo.yml
 [error] Errno::ENOENT : No such file or directory @ rb_sysopen - /kontena/test/foo.yml
         See /root/.kontena/kontena.log or run the command again with environment DEBUG=true set to see the full exception
```

### After
```
/kontena/test # kontena stack install foo.yml
 [error] RuntimeError : Not found: no such file foo.yml or invalid uri scheme
         See /root/.kontena/kontena.log or run the command again with environment DEBUG=true set to see the full exception
```